libavcodec/rkmppdec.c

```diff
@@ -515,7 +515,8 @@ static int rkmpp_export_frame(AVCodecContext *avctx, AVFrame *frame, MppFrame mp
         layer->format = rkmpp_get_drm_afbc_format(mpp_fmt);
         layer->nb_planes = 1;
         layer->planes[0].pitch = mpp_frame_get_hor_stride(mpp_frame) * 3 / 2;
-        layer->planes[0].offset = 0;
+        frame->crop_top = mpp_frame_get_offset_y(mpp_frame);
```

Comment:
We were too focused on `AVDRMFrameDescriptor` and ignored `AVFrame` itself 🤣

Reply:
Yeah, but i think there may be stiill one theoretical problem, if in future a decoder provides frames with multiple layers and with offset different on each layers then crop fields can not be used. But i am not sure if there is such a case, ie: nv12m format has multiple planes and they are represented ni multiple planes of a layer, not in different layer where there is one plane. In any case this is not todays problem so idont care :)